src/Components/Blazor/Server/src/Services/BlazorHostingServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net.Mime;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class BlazorHostingServiceCollectionExtensions
+    {
+        public static IServiceCollection AddBlazorStaticFilesConfiguration(this IServiceCollection services)
+        {
```

Comment:
When this method goes away in favour of `endpoints.MapClientSideBlazor`, is that where we will be able to plug in support for serving precompressed static resources for the May release?

I know you have more sophisticated thoughts about how to do it longer term, but I just want to check we also have a path to doing it via a cheaper mechanism in the short term.

Reply:
Yes, that's the idea.